docs/user-guides/backup-configuration.md

```diff
@@ -0,0 +1,248 @@
+| merit :+1:  | Easy to use<BR>Highly compatible with Kubernetes<BR>Complete with internal network<BR>Safety backup if applying CoW                                         | Easy to access backup files<BR>It fan be shared and used by multiple clusters                         | The safest of these methods                                                                                                                      |
+| demerit :x: | A bit hard to check backup files<BR>Can not share backup files for several Vald clusters                                                                    | Need to communicate with external network                                                             | Need to operate both storages<BR>The most expensive way                                                                                          |
+
+## Backup configuration
+
+This section shows the best practice for configuring backup features with PV, S3, or PV + S3.
+
+Each sample configuration yaml is published on [here](https://github.com/vdaas/vald/tree/master/charts/vald/values).
+Please refer it for more details.
+
+### General
+
+Regardless of the backup destination, the following Vald Agent settings must be set to enable backup.
+
+- `agent.ngt.index_path` is the location where those files are stored.
+- `agent.ngt.in-memory-mode=false` means storing index files in the local volume.
+
+In addition, `agent.terminationGracePeriodSeconds` value should be long enough to ensure the backup speed.
+
+```yaml
+agent:
+  ...
+  # We recommend setting this value long enough to ensure the backup speed of PV, since the Index is backed up at the end of the pod.
+  terminationGracePeriodSeconds: 3600
+  ngt:
+    ...
+    index_path: "/var/ngt/index"
+    enable_in_memory_mode: false
+    ...
+```
+
+### Persistent Volume
+
+#### requirement
+
+You must prepare PV before deployment when using Kubernetes Persistent Volume (PV) for backup storage.
+Please refer to the setup guide of the usage environment for the provisioning PV.
+
+For example:
+
+- [GKE setup PV document](https://cloud.google.com/kubernetes-engine/docs/concepts/persistent-volumes)
+- [EKS storage document](https://docs.aws.amazon.com/eks/latest/userguide/storage.html)
+
+#### configuration
+
+After provisioning PV, the following parameters are needed to be set.
+
+```yaml
+agent:
+  ...
+  persistentVolume:
+    # use PV flag
+    enabled: true
+    # accessMode for PV (please verify your environment)
+    accessMode: ReadWriteOncePod
+    # storage class for PV (please verify your environment)
+    storageClass: local-path
```

Comment:
I think it would be better to set GKE storage class and configuration as a sample.